opentrons-ai-client/src/components/organisms/Settings/FeatureFlag.tsx

```diff
@@ -0,0 +1,60 @@
+import { useTranslation } from 'react-i18next'
+
+import {
+  COLORS,
+  DIRECTION_COLUMN,
+  Flex,
+  JUSTIFY_SPACE_BETWEEN,
+  ListItem,
+  SPACING,
+  StyledText,
+} from '@opentrons/components'
+
+import { ToggleButton } from '../../../molecules/ToggleButton'
+
+interface FeatureFlagProps {
+  enablePDProtocolGeneration: boolean
+  onTogglePDProtocolGeneration: () => void
+}
+
+export function FeatureFlag({
+  enablePDProtocolGeneration,
+  onTogglePDProtocolGeneration,
+}: FeatureFlagProps): JSX.Element {
+  const { t } = useTranslation('protocol_generator')
+
+  return (
+    <Flex
+      flexDirection={DIRECTION_COLUMN}
+      gridGap={SPACING.spacing8}
+      height="100%"
+    >
+      <StyledText desktopStyle="bodyLargeSemiBold">
+        {t('feature_flags')}
+      </StyledText>
+      <ListItem
+        type="default"
+        padding={SPACING.spacing16}
+        justifyContent={JUSTIFY_SPACE_BETWEEN}
+        flexDirection={DIRECTION_COLUMN}
+        gridGap={SPACING.spacing16}
+      >
+        <Flex justifyContent={JUSTIFY_SPACE_BETWEEN}>
```

Comment:
![Screenshot 2025-06-27 at 10 44 37 AM](https://github.com/user-attachments/assets/bc1d211f-d3cb-421c-9196-3e3166642d69)

```suggestion
        <Flex justifyContent={JUSTIFY_SPACE_BETWEEN} alignItems={ALIGN_CENTER}>
```

Reply:
Thanks, changed